cypress/e2e/dashboard/chapters/chapter-administrator.cy.ts

```diff
@@ -0,0 +1,237 @@
+import { User } from '../../../../cypress.config';
+import { expectNoErrors } from '../../../support/util';
+
+const firstChapterId = 1;
+const secondChapterId = 2;
+
+describe('Chapter Administrator', () => {
+  let chapterRoles;
+  let instanceRoles;
+  let users;
+  before(() => {
+    cy.fixture('chapterRoles').then((r) => {
+      chapterRoles = r;
+    });
+    cy.fixture('instanceRoles').then((r) => {
+      instanceRoles = r;
+    });
+    cy.fixture('users').then((u) => {
+      users = u;
+    });
```

Comment:
Do `r` and `u` stand for the JSON files?

I'm wondering if it's good idea to change them to something descriptive

Reply:
In a way yes. It's what cypress passes from fixture file, to the `.then` method. I didn't spend long on thinking about their names. The most obvious `chapterRoles`, `instanceRoles`, `users` can't be used to be able to assign to these variables from the outer scope.

I guess I could do `.then((fixture) => {(...)`.

Reply:
I am happy with `fixture`, it hints that people should look up `.fixture` to know what is going on, and it's simple 👍

Reply:
Could you confirm whether to change it, or not? I'm slightly confused now.

Reply:
Sure, my intention was to change it, but I was worried that it would be over complicate something simple, so I resolved it. (something weird like `userRoleInJson` or my old suggestions)

I am happy with simple `.then((fixture) => {(...)` as it isn't complicated, and it's a good hint, if you want to add it, please do.

```suggestion
    cy.fixture('chapterRoles').then((fixture) => {
      chapterRoles = fixture;
    });
    cy.fixture('instanceRoles').then((fixture) => {
      instanceRoles = fixture;
    });
    cy.fixture('users').then((fixture) => {
      users = fixture;
    });
```